tests/wamr-test-suites/test_wamr.sh

```diff
@@ -15,7 +15,7 @@ function help()
     echo "test_wamr.sh [options]"
     echo "-c clean previous test results, not start test"
     echo "-s {suite_name} test only one suite (spec|wasi_certification|wamr_compiler)"
-    echo "-m set compile target of iwasm(x86_64|x86_32|armv7_vfp|thumbv7_vfp|riscv64_lp64d|riscv64_lp64|aarch64)"
+    echo "-m set compile target of iwasm(x86_64|x86_32|armv7|armv7_vfp|thumbv7|thumbv7_vfp|riscv32|riscv32_ilp32f|riscv32_ilp32d|riscv64|riscv64_lp64f|riscv64_lp64d|aarch64|aarch64_vfp)"
```

Comment:
Could we output the message in multiple lines?

Reply:
Please review again